Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM alpine:3.9
+FROM alpine:3.20 AS ssm-builder
+
+RUN apk add dpkg curl; \
+    curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb"; \
```

Comment:
Did you test the `aws-session-manager-plugin` from the alpine repo? https://pkgs.alpinelinux.org/package/edge/community/x86_64/aws-session-manager-plugin or is there a specific reason to install it manually?

Reply:
I hadn't seen this, since it's only available for `edge` – thanks for the hint!
Unfortunately this newer version didn't work in a non-interactive shell, hence I needed to add `expect` as well, see https://github.com/formunauts/drone-ecs-deploy-advanced/pull/1/commits/1cb2c56741f54e2ffd946674e216ba8b37f00d98. But then I switched to `ssm send-command` which does work non-interactively, see https://github.com/formunauts/drone-ecs-deploy-advanced/pull/1/commits/1c6eaa457d7c14c9b025ae3b63af0f0a354afd54 :slightly_smiling_face: